README.md

```diff
@@ -0,0 +1,47 @@
+<div align="center">
+    <picture>
+        <source srcset="logo here" media="(prefers-color-scheme: dark)">
+        <img src="logo here">
+    </picture>
+    <h1>Gusto Python Client</h1>
+        <p><strong>Payroll, HR, benefits. Simplified</strong></p>
+        <p>Secondary Description.</p>
+    <a href="https://docs.gusto.com/embedded-payroll/docs/introduction"><img src="https://img.shields.io/static/v1?label=Docs&message=API Ref&color=4c2cec&style=for-the-badge" /></a>
+    <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/License-MIT-blue.svg?style=for-the-badge" /></a>
+</div>
+
+> [!NOTE]  
+> Exciting news! Our first Python library beta release is here.
+
+## Authentication
+
+Gusto Embedded supports two types of tokens that are used throughout development: company-level tokens and system-level tokens.
+
+System access tokens allow you to carry out application level actions such as:
+* Creating a partner managed company
+* Viewing invoices
+
+Company access tokens allow you to carry out company level actions, such as:
+* Getting company employees
+* Running payroll for a company
+
+Each partner managed company has a single access token that cannot be used to access other companies. If you're on a version older than 2023-05-01, see [Upgrading to Strict Access](https://docs.gusto.com/embedded-payroll/docs/strict-access) for more information on upgrading to strict access.
```

Comment:
TODO: We can probably remove this section, we are only generating client libraries for v2024-04-01 and above for now